### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by bryanzakounity**
Edited changelog for:

## [2.8.0-exp.1] - 2021-03-31

[Delete any line or section that does not apply]

### Purpose of this PR

[JIRA issue. Desc of feature/change. Links to screenshots, design docs, user docs, etc. Remember reviewers may be outside your team, and not know your feature/area that should be explained more.]

### Testing status

[Explanation of what’s tested, how tested and existing or new automation tests. Can include manual testing by self and/or QA. Specify test plans. Rarely acceptable to have no testing.]

- [ ] Added an automated test
- [ ] Passed all automated tests
- [ ] Manually tested 

### Documentation status

[Overview of how documentation is affected by this change. If there is no effect on documentation, explain why. Otherwise, state which sections are changed and why.]

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Samples status

- [ ] Re-packed Extras~/CinemachineExamples.unitypackage

### Technical risk

[Overall product level assessment of risk of change. Need technical risk & halo effect.]

### Comments to reviewers

[Info per person for what to focus on, or historical info to understand who have previously reviewed and coverage. Help them get context.]

### Package version

[Justification for updating either the patch, minor, or major version according to the [semantic versioning](https://semver.org/spec/v2.0.0.html) rules]

- [ ] Updated package version

### CODE REVIEW

**Comment by bryanzakounity** [comment]
There was one entry I did not understand (but that I edited anyway).  Line #31 ("- Bugfix: Changing targets on a live vcam misbehaved.")  I might have changed the meaning of this one.

**Comment by mjmvisser** [comment]
> There was one entry I did not understand (but that I edited anyway). Line #31 ("- Bugfix: Changing targets on a live vcam misbehaved.") I might have changed the meaning of this one.

@glabute ☝️

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,34 +5,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Bugfix: 3rdPersonFollow stuttered when Z damping was set high.
+- Regression fix: CinemachineInputProvider stopped providing input.
+- Bugfix: Lens aspect and sensorSize were not updated when lens OverrideMode != None.
+- Bugfix: Changing targets on a live vcam misbehaved.
+- Bugfix: Framing transposer did not handle empty groups.
 - Bugfix: Interrupting a transition with InheritPosition enabled was broken.
-- Bugfix: Cinemachine3rdPersonFollow was not handling collision by default.
-- Bugfix: SaveDuringPlay saves only components that have the SaveDuringPlay attribute.
+- Bugfix: Cinemachine3rdPersonFollow did not handle collision by default.
+- Bugfix: SaveDuringPlay saved only components that have the SaveDuringPlay attribute.
```
Comment (glabute) [inline CHANGELOG.md:35]: SaveDuringPlay was saving some components that did not have the SaveDuringPlay attribute.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,34 +5,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Cinemachine3rdPersonFollow can operate without the physics module and without collision resolution.
+- Bugfix: 3rdPersonFollow collision resolution failed when the camera radius was large.
+- Bugfix: 3rdPersonFollow damping operated in world space instead of camera space.
+- Bugfix: 3rdPersonFollow stuttered when Z damping was set high.
+- Regression fix: CinemachineInputProvider stopped providing input.
+- Bugfix: Lens aspect and sensorSize were not updated when lens OverrideMode != None.
+- Bugfix: Changing targets on a live vcam misbehaved.
+- Bugfix: Framing transposer did not handle empty groups.
 - Bugfix: Interrupting a transition with InheritPosition enabled was broken.
-- Bugfix: Cinemachine3rdPersonFollow was not handling collision by default.
```
Comment (glabute) [inline CHANGELOG.md:34]: Bugfix: Cinemachine3rdPersonFollow was handling collision by default, now it is disabled by default.

**Review by glabute — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,34 +5,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
-- Switching targets (Follow, LookAt) is smooth by default. For the old behaviour, after changing the targets, set PreviousStateIsValid to false.
+- Switching targets (Follow, LookAt) is smooth by default. For the old behaviour, change the targets then set PreviousStateIsValid to false.
 - Bugfix: Reversing a blend in progress respects asymmetric blend times.
 - Regression fix: CmPostProcessing and CmVolumeSettings components setting Depth of Field did not work correctly with Framing Transposer. 
-- Regression fix: 3rdPersonFollow keeps player in view when Z damping is high
+- Regression fix: 3rdPersonFollow keept player in view when Z damping is high.
```
[the remark this answers is not in the document]
Comment (bryanzakounity) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (bryanzakounity) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (bryanzakounity) [reply CHANGELOG.md:34]: Done.

**Review by glabute — APPROVED** [review]

**Pull request merged**